src/main/java/com/flansmod/common/teams/TeamsRound.java

```diff
@@ -113,6 +114,22 @@ public Team getOtherTeam(Team team)
 		return teams[0];
 	}
 	
+	public Team getTeam(EntityPlayer player)
```

Comment:
Would be worth returning an Optional in this method instead of returning null and having possible issues down the line.